If CROSS_COMPILE is set, that means the user is trying to specify the
compiler manually. So don't error out when YOCTO_SDK_INSTALL_DIR isn't
set but CROSS_COMPILE is set.

Signed-off-by: Juro Bystricky <juro.bystricky@intel.com>
---
diff --git a/scripts/Makefile.toolchain.zephyr b/scripts/Makefile.toolchain.zephyr
index b964abf..69093f3 100644
--- a/scripts/Makefile.toolchain.zephyr
+++ b/scripts/Makefile.toolchain.zephyr
@@ -12,6 +12,7 @@
 #
 #######################################################################
 
+ifndef CROSS_COMPILE
 REQUIRED_SDK_VER=0.8.2
 
 ifndef ZEPHYR_SDK_INSTALL_DIR
@@ -25,6 +26,7 @@ ifeq ($(SDK_CHECK),1)
 $(error (The SDK version you are using is old, please update your SDK. You need at least SDK version $(REQUIRED_SDK_VER)))
 endif
 endif
+endif
 
 ifeq ($(HOST_OS),MINGW)
 	TOOLCHAIN_HOME = ${ZEPHYR_SDK_INSTALL_DIR}/sysroots/i686-pokysdk-mingw32
